md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/AppDeploymentQueueAPI.java

```diff
@@ -46,9 +45,201 @@
     private ProcessDAO processDAO;
     @Autowired
     private PropertiesDAO propertiesDAO;
+    @Autowired
+    private AppDeploymentQueueStatusDAO appDeploymentQueueStatusDAO;
+    @Autowired
+    private UsersDAO usersDAO;
+    @RequestMapping(value = {"/", ""}, method = RequestMethod.POST)
+    @ResponseBody
+    public
+    RestWrapper insert(@ModelAttribute("acq")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "insert" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)